common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -11,7 +11,16 @@
  */
 
 (function (requirejs, require, define) {
-
+require.config({
```

Comment:
I'm not comfortable with this as it seems like just another hack that doesn't get to the root of the problem:
1. Why is the RequireJS instance passed in here sometimes not configured correctly?
2. What happens in the situation where the require instance is the correctly configured one? Does this change reconfigure (and possibly break) the correct one?
3. How can we get a dynamically loaded moment.js to use the version from the cache?

It seems that the core question we need answered is number 1. Is there a race condition where RequireJS hasn't configured itself yet but the video player is starting to initialize? Can we move the RequireJS configuration earlier and enforce that the xmodules are only loaded once that has completed? Will that slow things down further? 

Note that this wouldn't help with loading the non-cached version of the file, so an alternate solution would be needed for that. We probably need something like Will's solution in this old PR where he looked up the MD5 hashed URL for each file and added it to the RequireJS paths:

https://github.com/edx/edx-platform/pull/6765/files

Another option that @cahrens suggested is to load moment.js into the global namespace and not use Require to access it. This is a reasonable alternative and would ensure that we get the correct cached version. It might be simpler to do this in the short term before we think through the better solution.